go/vt/vtgate/executor_dml_test.go

```diff
@@ -53,12 +55,8 @@ func TestUpdateEqual(t *testing.T) {
 		Sql:           "update `user` set a = 2 where id = 1",
 		BindVariables: map[string]*querypb.BindVariable{},
 	}}
-	if !reflect.DeepEqual(sbc1.Queries, wantQueries) {
-		t.Errorf("sbc1.Queries: %+v, want %+v\n", sbc1.Queries, wantQueries)
-	}
-	if sbc2.Queries != nil {
-		t.Errorf("sbc2.Queries: %+v, want nil\n", sbc2.Queries)
-	}
+	testQueries(t, sbc1, wantQueries)
```

Comment:
should we rename this to `assertQueries` or something? this method is just asserting that we got the right queries back, right?